### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by nguasch**
I've added Markup for forms component.
Not includes:
- Autocomplete (requires JS)
- field-group-grow (only works in `row-centered`)
- Captcha (It's an example)

Please, review @PablitoGS

### CODE REVIEW

File: styles/elements/_forms.scss [hunk 1]
```diff
@@ -133,6 +159,300 @@ textarea {
...
+                <input id="input-error" type="text" />
+                <p class="error-info">Lorem ipsum dolor sit amet.</p>
+            </li>
+            <li class="disabled">
+                <label for="input-disabled">Basic disabled field</label>
+                <input id="input-disabled" type="text" disabled />
+            </li>
+            <li>
+                <label for="input-PostalCode">Postal Code</label>
+                <input id="input-PostalCode" type="text" class="input-PostalCode" />
```
Comment (PablitoGS) [inline styles/elements/_forms.scss]: maxlength="5"

**Comment by PablitoGS** [comment]
Input group disabled

**Comment by nguasch** [comment]
done @PablitoGS

**Comment by PablitoGS** [comment]
:+1:

**Pull request merged**